### PR TITLE
Add support for logging in timezone other GMT

### PR DESCRIPTION
**Pull request by coderkd10**
Solves #4 

@sirrodgepodge please review this and lemme know if any changes are required.
Right now I using [`momentjs`](http://momentjs.com/timezone/) which looked the goto library for dealing with timezones in js.

### CODE REVIEW

**Comment by sirrodgepodge** [comment]
Just woke up, took a brief look, great idea!  Will look again and merge later today

**Comment by sirrodgepodge** [comment]
apologies on lateness, thank you for your work! 

Do you mind making a few changes to make sure we keep timezone and date format as separate ideas?

Making a PR now based off your code

**Comment by coderkd10** [comment]
@sirrodgepodge no issues , and I'll be happy to make any necessary changes 😄 .
Let me rephrase the requirement according to what I understand :
We want the user be able to independently specify timezone & date format. e.g,  user can specify `dateTimeFormat = iso` and `timezone = EST`, and we should display the time in iso format along with respecting the timezone supplied.
Is my understanding correct? 
If yes, then I can go ahead and add some commits to this PR to accomodate this requirement. 

(Since I didn't require other date formats, so I ignored them previously, and hacked up a quick solution for my specific requirement. But yes, this makes more sense in the general case.)

**Comment by sirrodgepodge** [comment]
https://github.com/sirrodgepodge/morgan-body/pull/6

^here I'm separating the two, reading your comment now

**Comment by sirrodgepodge** [comment]
Your understanding in your comment is correct.  The odd thing is that UTC is both a standard and a timezone (it's always GMT I believe), looking into this more, want to make sure we're 100% correct and don't look like jesters.

**Comment by sirrodgepodge** [comment]
okay, went bananas on this one :).

Merged in your PR here, but added a bunch of handling for timezones and dateTime formats here: https://github.com/sirrodgepodge/morgan-body/commit/0db6b031a519ea01851d2162bca24cedb8b00ff4

Also added tests to make the project a bit more real :), thank you for the push to do so
